InstantSearch.js/getting-started/src/app.js

```diff
@@ -1,8 +1,8 @@
 /* global instantsearch */
+import algoliasearch from 'algoliasearch/lite';
```

Comment:
if InstantSearch is global, this should probably be too?

Reply:
Yep!